README.md

```diff
@@ -371,3 +375,113 @@ Validation error codes:
 1005 - Invalid ECDSA or Hash
 1006 - Unsupported option
 ```
+
+## Websocket API
+
+### Orderbook
+
+#### Request
+
+```
+{
+    "type": "subscribe",
+    "channel": "orderbook",
+    "payload": {
+        "baseTokenAddress": "0x323b5d4c32345ced77393b3530b1eed0f346429d",
+        "quoteTokenAddress": "0xef7fff64389b814a946f3e92105513705ca6b990",
+        "snapshot": true,
+        "limit": 100
+    }
+}
+```
+
+- `baseTokenAddress` - address of token designated as the baseToken in the currency pair calculation of price (required).
+- `quoteTokenAddress` - address of token designated as the quoteToken in the currency pair calculation of price (required).
+- `snapshot` - if `true`, a snapshot of the orderbook will be sent before any updates to the orderbook.
+- `limit` - maximum number of bids and asks in orderbook snapshot.
+
```

Comment:
Add a comment about how `base` and `quote` params allow the Relayer to return the orders as `bids` and `asks`

Reply:
Also mention that by default `bids`/`asks` sorted to return those closest to market price first.